src/page-modules/contact/ticket-control/postpone-paymnet/postponePaymentForm.tsx

```diff
@@ -0,0 +1,152 @@
+import { FormEventHandler } from 'react';
+import { Button } from '@atb/components/button';
+import { Input } from '../../components/input';
+import { SectionCard } from '../../components/section-card';
+import { PageText, useTranslation } from '@atb/translations';
+import { useMachine } from '@xstate/react';
+import { formMachine } from './postponePaymentFormMachine';
+
+export const PostponePaymentForm = () => {
+  const { t } = useTranslation();
+  const [state, send] = useMachine(formMachine);
+
+  const onSubmit: FormEventHandler<HTMLFormElement> = async (e) => {
+    e.preventDefault();
+
+    if (!state.matches('submitting')) return;
+
+    const response = await fetch('/api/contact', {
+      method: 'POST',
+      body: JSON.stringify(state.context),
```

Comment:
API uses `firstName` and `lastName`, not `firstname` or `lastname`. Need to handle that here